packages/contracts/src/dollar/core/ERC1155Ubiquity.sol

```diff
@@ -20,8 +20,8 @@ contract ERC1155Ubiquity is ERC1155, ERC1155Burnable, ERC1155Pausable {
 
     IAccessControl public accessCtrl;
     // Mapping from account to operator approvals
-    mapping(address => uint256[]) private _holderBalances;
-    uint256 private _totalSupply;
+    mapping(address => uint256[]) public _holderBalances;
```

Comment:
Usually variables with an `_` prefix are meant to be private. Here we convert visibility from `private` to `public` so we don't need the `_` prefix anymore.

Reply:
> Usually variables with an `_` prefix are meant to be private. Here we convert visibility from `private` to `public` so we don't need the `_` prefix anymore.

sure

Reply:
> Usually variables with an `_` prefix are meant to be private. Here we convert visibility from `private` to `public` so we don't need the `_` prefix anymore.

there is a function called totalSupply which is causing problem cuz the variable above has the same name

Reply:
```function totalSupply() public view virtual returns (uint256) {
        return totalSupply;
    }

Reply:
perhaps we could keep the `_` and transform `public` to `internal` or simply remove the `totalSupply()` method because public state variables have a default getter